tests/core/test_cells.py

```diff
@@ -43,7 +43,6 @@
     CellType.QUAD,
     CellType.TETRA,
     CellType.VOXEL,
-    CellType.HEXAHEDRON,
 ]
 faces_types = [
```

Comment:
Although not relevant to this discussion, the variable faces_types is an unused variable and should be removed or a test added.

Reply:
Done.  Good catch, might as well clean up wherever possible here.